completion/available/hub.completion.bash

```diff
@@ -62,7 +64,7 @@ if _is_function _git && ! _is_function __git_list_all_commands_without_hub; then
 	# hub alias [-s] [SHELL]
 	_git_alias() {
 		local i c=2 s=-s sh shells="bash zsh sh ksh csh fish"
-		while [ $c -lt $cword ]; do
+		while [ $c -lt "$cword" ]; do
```

Comment:
```suggestion
		while [ "$c" -lt "$cword" ]; do
```

or

```suggestion
		while [[ $c -lt $cword ]]; do
```

or

```suggestion
		while ((c < cword)); do
```

The pattern `[[ unquoted == quoted ]]` is only used for the glob pattern matching (`==` and `!=`) in `[[`.